### ISSUE TITLE
Unclear structure for Town Halls and After Town Halls

### ISSUE THREAD

**Issue by VanCardui** [Catalyst-Swarm/Intersect-Town-Hall#24]
### Source: 
Town Hall #1, Thurs 6th June 2024

### Problem:
There is no agreed structure for either Town Halls or After-Town Halls.

**Comment by VanCardui** [comment]
### Current situation:
Currently, there is no specific structure for Intersect Town Halls or After THs. There are patterns that have been defined in *other* THs in other ecosystems, but they might not suit *this* ecostsyem.

### How we want the current situation to change:
We would like to know what people expect Intersect TH to be like, and structure it accordingly. 

### Who are the stakeholders?
-

### Any previous attempts to address this problem?
- 

### Questions and possible approaches:
- Intersect members might not know yet what they expect or want - so perhaps it's right to start it, and then change it if needed.
- Will defining a **structure** help us get closer to what TH "should" be like? Will a structure make it easier for community members to engage with the space?
- If yes, how rigid/flexible should that stucture be? What should it consist of?

### Ideas:
- **Suggestion from Tevo**: _Town Halls are usually expected to be spaces to build strong community connections and get relevant updates to empower and inspire members. It's also a place where we can celebrate our collective achievements._ 
- **Sugestion from Vani:** _And a place where people share community--held IP and it gets documented_
- **Do we need to ask the community** what they think it should be like before we start? Or should we work with these suggestions as a starter?

**Comment by stephen-rowan** [comment]
Abe - Who is designing the process / driving the train.
Include pre planned activities

**Comment by stephen-rowan** [comment]
Samuel - Here is a link to a google drive me and some others are working out of to compile materials for better communication governance to the community:

https://drive.google.com/drive/folders/1yXDofZVRZf0jOGTiy9uLO3y0zOzAL_Gm?usp=sharing

I have a discord announcement template in there as a Google Doc ready to roll out to our discord communities. Need that Voting stuff handed out to the community ASAP it starts today

**Comment by VanCardui** [comment]
We have defined a broad structure, as shown in the slides we used at TH on 27th June. https://docs.google.com/presentation/d/1RURCxhUYcfNtGjXe2G_N1VXJ53Zifj-Ib9IqRD3EFNg/edit?usp=sharing (Note - this slide design can no longer be used - Intersect staff have said we need to use theirs - see https://github.com/Catalyst-Swarm/Intersect-Town-Hall/issues/31)

The structure is also reflected in the agenda - see https://github.com/Catalyst-Swarm/Intersect-Town-Hall/issues/32

How well did it work?
How can we refine it?

**Comment by VanCardui** [comment]
After 2 THs (June and July) the structure has been firmed up, and so far, it seems to fit what is needed. 

We have transferred our slides to the Intersect team's required design: see https://docs.google.com/presentation/d/12oZ7jSuTko34hvfxY4bzVO0ygmutxNTUPfjJ-Qq7mIg/edit?usp=sharing

The structure is:

1. an introduction, explaining what TH and ATH are, for new people
2. an interactive element with a live vote on a topic (this was suggested in an early planning session by community members, so we're keen to keep it. The idea was a bit woolly in June, but we have now firmed it up and it seems to be working quite well.)
3. introducing the problem board and feeding back about planning sessions
4. space for updates from working groups and special interest groups (groups are able to send an update, even if they do not attend)
5. space for updates from the Intersect staff team
6. Human Interfacing (short breakout rooms to get people talking to each other)
7. After TH

We will use our planned survey to find out if there are other elements that people want to see in a TH.
